xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceHmacDigest.java

```diff
@@ -69,23 +60,6 @@ public String digestParams(RestInvocation restInvocation) {
       Mac mac = getMac();
       mac.update(input.getBytes("UTF-8"));
```

Comment:
Use standardcharsets.utf8 instead of this string

Reply:
done